packages/server/graphql/types/RemoveMattermostAuthPayload.ts

```diff
@@ -0,0 +1,25 @@
+import {GraphQLID, GraphQLObjectType} from 'graphql'
+import StandardMutationError from './StandardMutationError'
+import User from './User'
+import {GQLContext} from '../graphql'
+
+const RemoveMattermostAuthPayload = new GraphQLObjectType<any, GQLContext>({
```

Comment:
let's use the new Success | Error pattern that is used here:  https://github.com/ParabolInc/parabol/blob/f49e74c0ca6a56a6ae13b385f094d434fd0ed6df/packages/server/graphql/types/AddMattermostAuthPayload.ts